### DISCUSSION TITLE
Is there a way to customize the location where the snapshot files generate?

### DISCUSSION THREAD

**Discussion by hyfdev** [mitsuhiko/insta#337] [Q&A] [answered]
It would be great we could customize the location.

**Comment by mitsuhiko** [comment, marked as answer]
You can by reconfiguring the settings: https://docs.rs/insta/latest/insta/struct.Settings.html#method.set_snapshot_path

See also: https://insta.rs/docs/settings/

**Reply by hyfdev** [reply]
Thank you. I miss it.
